src/stylesheets/elements/form-controls/_range-input.scss

```diff
@@ -18,6 +18,10 @@
   border: none;
   box-shadow: 0 0 0 units($theme-input-select-border-width) color("base");
   cursor: pointer;
+
+  @media (forced-colors: active) {
+    outline: 2px solid transparent;
+  }
```

Comment:
# Range

This adds an outline to the range slider handle (or "thumb") when high contrast mode is detected

Currently:

![image](https://user-images.githubusercontent.com/25211650/153293313-74e3d365-5cec-43e4-a6c2-248a509cddee.png)

After Fix:

![image](https://user-images.githubusercontent.com/25211650/153293371-6263bafe-b739-4120-8f54-b01eb55c2ad0.png)